### PR TITLE
Keros

### PR DESCRIPTION
**Pull request by stevenernst131**
Accessibility Fixes. Mostly color changes to comply with contrast rules.

### CODE REVIEW

File: AngularApp/src/app/shared/components/toggle-button/toggle-button.component.html [hunk 1]
```diff
@@ -4,10 +4,10 @@
     </label>
     <div class="setting-control-container">
       <div role="radiogroup" class="medium" aria-disabled="false">
-        <div role="radio" (click)="update(false)" [attr.aria-checked]="!selected">
+        <div role="radio" (click)="update(false)" attr.aria-checked="{{!selected ? 'true': 'false'}}">
```
Comment (cindylovescoding) [inline AngularApp/src/app/shared/components/toggle-button/toggle-button.component.html]: What's the difference of this with the previous code: [attr.aria-checked]="!selected" ?

Comment (stevenernst131) [reply AngularApp/src/app/shared/components/toggle-button/toggle-button.component.html]: Good catch, I need to change them both to match the second version

File: AngularApp/package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "@angular/upgrade": "5.2.7",
     "@swimlane/ngx-datatable": "^11.3.2",
     "@types/applicationinsights-js": "^1.0.5",
-    "applens-diagnostics": "0.0.17",
+    "applens-diagnostics": "file:///D:\\applensv3\\AngularApp\\dist.tgz",
```
Comment (stevenernst131) [inline AngularApp/package.json]: I need to revert this back to the standard package

File: AngularApp/src/app/app.module.ts [hunk 3]
```diff
@@ -34,25 +34,29 @@ import { GenericCommsService } from './shared/services/generic-comms.service';
...
-        path: 'subscriptions/:subscriptionid/resourcegroups/:resourcegroup/providers/microsoft.web/sites/:resourcename/slots/:slot',
-        loadChildren: 'app/resources/web-sites/web-sites.module#WebSitesModule'
-      },
-      {
-        path: 'subscriptions/:subscriptionid/resourcegroups/:resourcegroup/providers/microsoft.web/hostingenvironments/:resourcename',
-        loadChildren: 'app/resources/web-hosting-environments/web-hosting-environments.module#WebHostingEnvironmentsModule'
+        path: 'resource',
+        loadChildren: 'app/resources/resources.module#ResourcesModule'
       }
+      // {
```
Comment (stevenernst131) [inline AngularApp/src/app/app.module.ts]: will remove comments

File: AngularApp/src/app/resources/web-sites/services/site-feature.service.ts [hunk 4]
```diff
@@ -153,7 +152,7 @@ export class SiteFeatureService extends FeatureService {
           description: '',
           featureType: FeatureTypes.Tool,
           clickAction: this._createFeatureAction(ToolNames.AutoHealing, 'Diagnostic Tools', () => {
-            this._router.navigateByUrl(`${resourceId}/tools/mitigate`);
+            this._router.navigateByUrl(`resource${resourceId}/tools/mitigate`);
```
Comment (cindylovescoding) [inline AngularApp/src/app/resources/web-sites/services/site-feature.service.ts]: Do you need a slash between?

**Pull request merged**